packages/cli/generators/project/templates/.vscode/launch.json

```diff
@@ -0,0 +1,38 @@
+{
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Launch Program",
+      "skipFiles": [
+        "<node_internals>/**"
+      ],
+      "program": "${workspaceFolder}/index.js",
```

Comment:
By default, we will have `dist/index.js` instead of root level `index.js` for applications generated by `lb4`.

Reply:
Good catch. I verified with a new app, `${workspaceFolder}/dist/index.js` is correct 👍